### PR TITLE
Yy feature fixbrowsepage

### PR DESCRIPTION
**Pull request by yenyin262**
-fix browse page functionality

### CODE REVIEW

**Review by piahmed — CHANGES_REQUESTED** [review]
@yenyin262 There are other changes in this PR. Please update. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,60 +1,50 @@
-## Shop This City
+# Shop This City
+
   A mobile friendly application build on iOS Platform using React Native. It allows Users to favourite items from assorted well-known brands by their categories which enables the User to build up their own Pinterest Board. A divine feature that allows a User to go back to the items easily when the item has not been purchased.
```
Comment (zareef4) [inline README.md]: Please redo this merge... it looks like you got rid of some important things? 

The screenshots are gone and some sections were deleted

If it was intentional ignore this comment.

File: client/js/screens/Browse/styles.js [hunk 2]
```diff
@@ -1,21 +0,0 @@
-import {StyleSheet, Dimensions} from 'react-native';
```
Comment (zareef4) [inline client/js/screens/Browse/styles.js]: was this deliberate? to get rid on the styles file?

File: README.md (hunk 1 above)
Comment (yenyin262) [reply README.md]: yup i made an animated gif to show the pictures :)

File: client/js/screens/Browse/styles.js (hunk 2 above)
Comment (yenyin262) [reply client/js/screens/Browse/styles.js]: yes

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]
Please resolve merge conflicts. 
Thanks

**Review by siddharthparmar7 — APPROVED** [review]

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts.

**Review by iyuvraajsingh — APPROVED** [review]

**Review by piahmed — DISMISSED** [review]
Approving to merge.

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]
Please fix the merge conflicts.
Thanks

**Pull request merged**